paddle/fluid/framework/operator.cc

```diff
@@ -1061,7 +1061,8 @@ bool OperatorWithKernel::SupportsMKLDNN(
 bool OperatorWithKernel::CanMKLDNNBeUsed(const framework::ExecutionContext& ctx,
                                          proto::VarType::Type data_type) const {
   bool use_mkldnn_ctx =
-      ctx.Attr<bool>("use_mkldnn") && platform::is_cpu_place(ctx.GetPlace());
+      (ctx.Attr<bool>("use_mkldnn") || data_type == proto::VarType::BF16) &&
```

Comment:
Why this change is needed?

Reply:
Must have left it during experiments, thanks.